src/components/ProjetoDialog.tsx

```
@@ -23,6 +26,30 @@ export function ProjetoDialog({
   onOpenChange,
 }: ProjetoDialogProps) {
   const [imgSrc, setImgSrc] = React.useState<string>(FormPng)
+  const [email, setEmail] = useState<string | null>(null)
+  const [roles, setRoles] = useState<"ROLE_SERVIDOR" | "ROLE_ALUNO" | null>(null)
```

ESLint

[error] Replace `"ROLE_SERVIDOR"·|·"ROLE_ALUNO"·|·null>(null` with `'ROLE_SERVIDOR'·|·'ROLE_ALUNO'·|·null>(⏎····null,⏎··`

(prettier/prettier)